examples/ibm-iks-openshift/README.md

```diff
@@ -1,24 +1,141 @@
-# IBM cluster with openshift example
+# Example for IKS openshift resources
 
 This example shows how to create a Kubernetes Cluster with openshift.
-To run, configure your IBM Cloud provider
 
-Running the example
+Following types of resources are supported:
 
-For planning phase
+* [ Container Cluster resource ](https://ibm-cloud.github.io/tf-ibm-docs/v0.17.3/r/container_cluster.html)
+* [ Container Cluster terraform doc](https://cloud.ibm.com/docs/terraform?topic=terraform-container-resources)
 
-```shell
-terraform plan
+
+## Terraform versions
+
+Terraform 0.12. Pin module version to `~> v1.5.1`. Branch - `master`.
+
+Terraform 0.11. Pin module version to `~> v0.27.0`. Branch - `terraform_v0.11.x`.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+Run `terraform destroy` when you don't need these resources.
+
+## Example Usage
+
+In the following example, you can create a Kubernetes cluster with a default worker pool with one worker:
+
+```hcl
+resource "ibm_container_cluster" "testacc_cluster" {
+  name            = "test"
+  datacenter      = "dal10"
+  machine_type    = "free"
+  hardware        = "shared"
+  public_vlan_id  = "vlan"
+  private_vlan_id = "vlan"
+  subnet_id       = ["1154643"]
+
+  default_pool_size = 1
+
+  webhook {
+    level = "Normal"
+    type  = "slack"
+    url   = "https://hooks.slack.com/services/yt7rebjhgh2r4rd44fjk"
+  }
+}
 ```
 
-For apply phase
+Create the Kubernetes cluster with a default worker pool with 2 workers and one standalone worker as mentioned by worker_num:
```

Comment:
Remove this section
Create the Kubernetes cluster with a default worker pool with 2 workers and one standalone worker as mentioned by worker_num:We deprected worker_num